### ISSUE TITLE
Update from 0.7.1 to 0.9.0 - Message to update when i'm connected

### ISSUE THREAD

**Issue by Draleg** [shaarli/Shaarli#873]
Hello all,
I just do the update frome 0.7.1 to 0.9.0 .
I backup my data/datastore.php, and copy/past all the package of the 0.9.0 .

Since this update I have a message on the top right corner to do an update:
"● Shaarli 0.7.0 */ is available."
I don't think it was normal lol...

How can I check if the update will do correctly ?

I do the copy/past two times, but the message stay (when I'm logged).

Thank's in advance.

**Comment by virtualtam** [comment]
Hi @Draleg!

Could you check the following configuration section under `data/config.json.php`:

```json
[...]
    "updates": {                                                                                                                                                                                                    
        "check_updates": false,
        "check_updates_branch": "latest",
        "check_updates_interval": 86400
    },
[...]
```

As well as the contents of the `shaarli_version.php` file?

See https://github.com/shaarli/Shaarli/wiki/Shaarli-configuration for more details regarding available settings

**Comment by ilesinge** [comment]
Hello ! I had the exact same bug after upgrading to 0.9, and reading the latest commits on the 0.9 branch, I tried to apply this change : https://github.com/shaarli/Shaarli/commit/c318096c7a6fb3f6b00bd8c694ab7acb8fbb7cd0
The upgrade prompt bug disappeared.

I can't reproduce the bug on my laptop though, it may be related to the PHP version used (5.6 on my server, 7.0 locally)

**Comment by Draleg** [comment]
Hello all,
Thank's for your reactivity, it was impossible to me before today to make test.

Thank's to ilesinge, it was the good commit ! 
I do this [modification](https://github.com/shaarli/Shaarli/commit/c318096c7a6fb3f6b00bd8c694ab7acb8fbb7cd0) and all works fine!

Have a nice day all.

**Comment by arnaudmm** [comment]
Had same issue, fixed it with @ilesinge comment

**Comment by MicroJoe** [comment]
Just had this bug too, and issue fixed with same modification as above.
![shaarli_versions](https://cloud.githubusercontent.com/assets/989521/26765576/0cfb6810-497f-11e7-83f2-de43a1682620.png)

This has to be fixed on the v0.9.0 stable version (or release a v0.9.1 with the fix).

**Comment by ilpssun** [comment]
Same issue here.  When I upgraded from 0.8.3 to 0.9.0 it announced erroneously that there was an upgrade available.  Above mentioned change fixed it.

**Comment by nodiscc** [comment]
I've just installed a fresh Shaarli instance in a VM using [these ansible tasks](https://github.com/nodiscc/srv01/blob/master/roles/webapp-shaarli/tasks/shaarli.yml) and I'm greeted with the message

`Shaarli 0.7.0 */ is available.`

So  this bug is not exclusively related to the upgrade procedure.

**Comment by virtualtam** [comment]
Closing as OP has marked the issue as solved, and a fix has been merged on the `0.9` branch
